src/scroll-timeline-base.js

```diff
@@ -370,7 +370,9 @@ export function addAnimation(scrollTimeline, animation, tickAnimation) {
     animation: animation,
     tickAnimation: tickAnimation
   });
-  updateInternal(scrollTimeline);
+  queueMicrotask(() => {
```

Comment:
I think if the timeline is active we expect the animation to tick immediately. It's only if this is a newly created timeline that hasn't ticked yet that we would expect it to wait until the next frame update to tick.

Reply:
Though I think we also expect the animation to not start immediately due to the auto aligning start time behavior, so perhaps I'm worried about an edge case that isn't really web visible. We can revisit if this causes issues but it seems to be a net plus on tests.